Demo/Demo/DemoMerchantAPI.swift

```diff
@@ -0,0 +1,78 @@
+import Foundation
+
+final class DemoMerchantAPI {
+
+    static let sharedService = DemoMerchantAPI()
+
+    private init() {}
+
+    func createOrder(orderParams: CreateOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
+        // TODO: get environment from settings in Demo app
+        guard let url = URL(string: DemoSettings.Environment.sandbox.baseURL + "/order?countryCode=US") else {
+            completion(.failure(.invalidURL))
+            return
+        }
+
+        var urlRequest = URLRequest(url: url)
+        urlRequest.httpMethod = "POST"
+
+        let encoder = JSONEncoder()
+        encoder.keyEncodingStrategy = .convertToSnakeCase
+        urlRequest.httpBody = try? encoder.encode(orderParams)
+        urlRequest.addValue("application/json", forHTTPHeaderField: "Content-Type")
+
+        URLSession.shared.dataTask(with: urlRequest) { data, response, error in
+            guard let data = data, error == nil else {
+                completion(.failure(.networkConnectionError))
+                return
+            }
+
+            if response == nil {
+                completion(.failure(.serverError))
+                return
+            }
+
+            do {
+                let order = try JSONDecoder().decode(Order.self, from: data)
+                completion(.success(order))
+            } catch {
+                completion(.failure(.dataParsingError))
+            }
+        }
+        .resume()
+    }
+
+    func processOrder(processOrderParams: ProcessOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
```

Comment:
Same note here - would be nice to have a line of doc just describing what happens in this function, making clear this isn't part of the SDK flow

Reply:
```suggestion
    /// Sends order to sample server for authorize/capture
    func processOrder(processOrderParams: ProcessOrderParams, completion: @escaping ((Result<Order, URLResponseError>) -> Void)) {
```

Does this clarify?

Reply:
Yep!